src/providers/dataProvider/index.test.ts

```diff
@@ -72,7 +72,7 @@ describe('generateBatchId', () => {
     it('should return 00', async () => {
       provider.clear(constants.R_BATCHES)
       const result = await generateBatchId(provider, year)
-      expect(result).toBe('00')
+      expect(result).toBe(`0${id}`)
```

Comment:
I can see you have changed these since the `id` changes.  But, I do want the results to be `00`, `01` and `02` - since I/we can see that they are the results that should be generated.

I _think_ you just have to initialise the `id` at the start of the test.

Hmm, since we want to reset the `id` _AND_ clear the batches, they could be in a jest `beforeAll` method.